sigs.yaml

```diff
@@ -49,7 +49,7 @@ sigs:
   - name: Architecture
     dir: sig-architecture
     mission_statement: >
-      The Architecture SIG maintains and evolves the design principles of Kubernetes, and provides a consistent 
+      The Architecture SIG maintains and evolves the design principles of Kubernetes, and provides a consistent
```

Comment:
:+1: 

This will lead to `sig-architecture/README.md` being changed after running `make`, right? I was just confused by this when I ran `make` for #848. Thanks for fixing this, @cblecker !